.github/dependabot.yaml

```diff
@@ -0,0 +1,58 @@
+version: 2
+updates:
+  - package-ecosystem: gomod
+    directory: /pkg/otlp/metrics
+    labels:
+      - dependencies
+    schedule:
+      interval: weekly
```

Comment:
Can we manually trigger one run of the bot ?

Reply:
Once this is merged we can trigger a run but it's not possible before merging this. For reference, it is a bit hidden, you have to navigate to Insights > Dependency graph > Dependabot > Click on "Last checked ..." > Click on "Check for updates". 

I can trigger a run after we merge this and see if there are any issues.